<launch>

    <include file="$(find course_agv_slam_task)/launch/extraction.launch"/>
    <include file="$(find course_agv_slam_task)/launch/icp_lm.launch"/>

	<node pkg="course_agv_slam_task" type="ekf" name="ekf" output="screen" >
		
		<!--from a rosbag-->
		<param name="robot_x" value="0"/>
		<param name="robot_y" value="0"/>
		<param name="robot_theta" value="0"/>

		<param name="match_th" value="0.5"/>
		<param name="landMark_num" value="0"/>
		<param name="noise_motion" value="0.3"/>
		<param name="noise_measure" value="0.1"/>
	</node>

</launch>
